### PR TITLE
docs(jokes): Add missing highlight of updated lines

### PR DESCRIPTION
**Pull request by dan-gamble**

### CODE REVIEW

File: docs/tutorials/jokes.md [hunk 1]
```diff
@@ -3952,7 +3952,7 @@ Remember that the `app/root.tsx` module is responsible for rendering our `<html>
 
 <summary>app/root.tsx</summary>
 
-```tsx filename=app/root.tsx lines=[57-67]
+```tsx filename=app/root.tsx lines=[27-47,59-55,57-67]
```
Comment (machour) [inline docs/tutorials/jokes.md:3955]: 59-55 looks wrong

**Comment by machour** [comment]
Hi @dan-gamble, converted the PR to draft since you're still making commits.
Let us know when you're finished 🙏🏼

**Comment by dan-gamble** [comment]
Just things I noticed as I was working my way through the tutorial. I've finished that now so this is ready 😅

**Review by machour — APPROVED** [review]

**Comment by machour** [comment]
Thank you so much @dan-gamble 🙏🏼

**Pull request merged**